modules/ROOT/pages/basic-authentication-ldap-concept.adoc

```diff
@@ -7,42 +7,76 @@ endif::[]
 |===
 >s| Policy Name | Basic Authentication: LDAP
 >s|Summary      | Allows access based on the basic authorization mechanism, with user-password defined on LDAP
->s|Category |
+>s|Category | Security
 >s|First Mule version available | 4.0.0
 .1+>.^s| Returned Status Codes
-|
+|401 - Unauthorized or invalid client application credentials
 |===
 
-The Lightweight Directory Access Protocol (LDAP) authentication policy establishes the configuration details for an Open LDAP or Active Directory LDAP that you set up for your enterprise. This policy is available only to an API you are managing in Mule 4 or later.
+The Lightweight Directory Access Protocol (LDAP) authentication policy specifies how an API with an Open LDAP or Active Directory LDAP mechanism must be authenticated. This policy is available only in Mule 4 or later. 
```

Comment:
"The Lightweight Directory Access Protocol (LDAP) authentication policy specifies how an API with an Open LDAP or Active Directory LDAP mechanism must be authenticated" -> "The Lightweight Directory Access Protocol (LDAP) authentication policy specifies how to restrict access to an API using LDAP authentication mechanism" or similar.

The API does not have the LDAP stuff. The policy calls LDAP to validate the info provided using the Basic Auth mechanism (ie Authorization HTTP header).

Reply:
Done! Thanks!